crates/ruma-client-api/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ Breaking changes:
 - The `content_disposition` fields of `media::get_content::v3::Response` and
   `media::get_content_as_filename::v3::Response` use now the strongly typed
   `ContentDisposition` instead of strings.
+- Change types of `SyncRequestListFilters::{room_types,not_room_types}` to
```

Comment:
This is not a breaking change. The sync::v4 module is behind an `unstable-*` cargo feature.